eng/docker-compose/start-local-dms.ps1

```diff
@@ -23,7 +23,11 @@ param (
 
     # Enable KafkaUI and OpenSearch Dashboard
     [Switch]
-    $EnableOpenSearchUI
+    $EnableOpenSearchUI,
+
+    # Enforce Authorization
+    [Switch]
+    $EnforceAuthorization
```

Comment:
I like this. Can we set an environment variable that would override the .env file setting? [Environment variables precedence in Docker Compose](https://docs.docker.com/compose/how-tos/environment-variables/envvars-precedence/):

> The order of precedence (highest to lowest) is as follows:
> 1. Set using [docker compose run -e in the CLI](https://docs.docker.com/compose/how-tos/environment-variables/set-environment-variables/#set-environment-variables-with-docker-compose-run---env).
> 2. Set with either the environment or env_file attribute but with the value interpolated from your [shell](https://docs.docker.com/compose/how-tos/environment-variables/variable-interpolation/#substitute-from-the-shell) or an environment file. (either your default [.env file](https://docs.docker.com/compose/how-tos/environment-variables/variable-interpolation/#env-file), or with the [--env-file argument](https://docs.docker.com/compose/how-tos/environment-variables/variable-interpolation/#substitute-with---env-file) in the CLI).

I think this means we can add `-e IDENTITY_ENFORCE_AUTHORIZATION=true` to the `docker compose` command below when `$EnforceAuthorization -eq $true`